schema/common/defaults/configuration/roles.yml

```diff
@@ -0,0 +1,54 @@
+---
+kind: configuration/roles
+title: "Roles to be enabled/disabled"
+name: default
+specification:
+  roles:
```

Comment:
How about simplifying things and use Ansible role names everywhere without features (e.g. `image_registry` instead of `image-registry`)?